test/cli.py

```diff
@@ -261,7 +261,7 @@ def test_003_cli_build_dir(self):
 
         assert_success_of_builds(test_scripts, cache_dir)
 
-    def test_021_cli_report(self):
+    def test_004_cli_report(self):
```

Comment:
This is one of the longest and most complex tests, so it is good to run it last. That's why it was number 21. Suggest moving it to the end of the file if you want all tests to be in index order.

Reply:
Moved large test to bottom!